code/modules/mob/living/carbon/human/spawners.dm

```diff
@@ -82,21 +88,16 @@
 				if(O)
 					O.take_external_damage(damage[limb])
 
-		if(damage["impale"] && ispath(damage["impale"]))
+		if(damage["impale"])
 			var/turf/T = H.near_wall(dir,2)
-			var/obj/item/weapon/arrow/rod/R = new(H)
 
 			if(T)
+				var/obj/item/weapon/arrow/rod/R = new(H)
+				H.set_dir(GLOB.reverse_dir[dir])
 				H.loc = T
 				H.anchored = 1
 				H.pinned += R
 
-	if(viruses)
```

Comment:
What's with these changes?

Reply:
Cleaned up some of this code I wrote years ago while I was fixing it. Nothing used this and it was a bit pointlessly specific like the rest of the spawner code.